go.mod

```diff
@@ -0,0 +1,17 @@
+module github.com/nabla-containers/runnc
+
+go 1.12
```

Comment:
This feels a little troubling.. is there a reason for locking in a golang version here? Anyway to do a version range?

Reply:
I'll look into the version lock of go mod. 1.11 didn't add it for some reason but building with 1.12 does.

Reply:
I checked on gopher slack, this version currently does not have any impact on the build. It's there for future use cases.